### PR TITLE
Enable Site-Isolation by default

### PR DESCRIPTION
**Pull request by jumde**

### CODE REVIEW

**Comment by diracdeltas** [comment]
is there a way to test this, other than just trying to print a page with cross-site iframes?

**Comment by diracdeltas** [comment]
per @bridiver comments in Slack, we should release this as off-by-default with a preference in about:preferences#security

File: patches/master_patch.patch [hunk 1]
```diff
@@ -1583,6 +1583,19 @@ index e54e2c0fcc8eac6fe637025c3c71f3d6834a4892..d817176d073d8a96fa76b944f6fb198d
...
  // Details about a redirection of a resource request.
+diff --git a/content/public/common/content_features.cc b/content/public/common/content_features.cc
+index f11d51a320cd3b1ddda4824bde4e29cc93fcd21d..6fe9a7b27f0254c776e493eded13f2364e305a16 100644
+--- a/content/public/common/content_features.cc
++++ b/content/public/common/content_features.cc
+@@ -300,7 +300,7 @@ const base::Feature kSignInProcessIsolation{"sign-in-process-isolation",
+ // Alternative to switches::kSitePerProcess, for turning on full site isolation.
+ // Launch bug: https://crbug.com/739418.
+ const base::Feature kSitePerProcess{"site-per-process",
+-                                    base::FEATURE_DISABLED_BY_DEFAULT};
```
Comment (bridiver) [inline patches/master_patch.patch]: this doesn't need a patch, it can be changed in atom_browser_main_parts

**Comment by diracdeltas** [comment]
i think this can be superseded by https://github.com/brave/browser-laptop/pull/12491 for the 0.19.x hotfix. if things look good we can then enable site isolation by default.

**Pull request closed without merging**